### PR TITLE
MAINT: remove duplicate eigenfaces header

### PR DESCRIPTION
**Pull request by mdhaber**
Closes gh-560

I removed the header from the index rather than `plot_eigenfaces.py` in case `plot_eigenfaces.py` is accessed as a stand-alone document. Need to check with the sidebar thing works properly.

### CODE REVIEW

**Review by mdhaber — COMMENTED** [review]
I don't think this should be merged as-is because it doesn't seem like the links are working as they're supposed to.

Instead, I wonder if this should even be a section of `index.rst`?  It is an example, so it appears with the other examples at the end of the page. Why don't we just remove it from here?

File: packages/scikit-learn/examples/plot_eigenfaces.py [hunk 1]
```diff
@@ -2,6 +2,13 @@
 The eigenfaces example: chaining PCA and SVMs
 =============================================
 
+
+.. sidebar:: Code and notebook
+
+   Python code and Jupyter notebook for this section are found
+   :ref:`here
+   <sphx_glr_packages_scikit-learn_auto_examples_plot_eigenfaces.py>`
+
```
Comment (mdhaber) [inline packages/scikit-learn/examples/plot_eigenfaces.py:11]:
```suggestion
```

File: packages/scikit-learn/index.rst [hunk 2]
```diff
@@ -1358,14 +1358,8 @@ of digits eventhough it had no access to the class information.
...
-.. sidebar:: Code and notebook
-
-   Python code and Jupyter notebook for this section are found
-   :ref:`here
-   <sphx_glr_packages_scikit-learn_auto_examples_plot_eigenfaces.py>`
+.. The eigenfaces example: chaining PCA and SVMs
+.. =============================================
 
 
 .. include:: auto_examples/plot_eigenfaces.rst
```
Comment (mdhaber) [inline packages/scikit-learn/index.rst:1365]:
```suggestion
```
(We would delete the remaining lines, too.)

**Review by mdhaber — COMMENTED** [review]
I don't think this should be merged as-is because it doesn't seem like the links are working as they're supposed to.

Instead, I wonder if this should even be a section of index.rst? It is an example, so it appears with the other examples at the end of the page. Why don't we just remove it from here?

**Review by jarrodmillman — APPROVED** [review]

**Pull request merged**